rclc/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ if(NOT CMAKE_CXX_STANDARD)
 endif()
 
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-  add_compile_options(-Wall -Wextra -Wpedantic)
```

Comment:
Do not remove pedantic flags

Reply:
agree. instead of avoiding the warning, we want to suppress the warning with correct code.

Reply:
I also agree, pedantic compiler flags should not be removed.

Reply:
Agree. Actually it's automatically generated in the CMakeLists.txt when we use `ros2 pkg create`

Reply:
Fixed (removed)